tests/dat/actions/unicode.tests/rust-1.34.txt

```diff
@@ -0,0 +1,23 @@
+extern crate serde_json;
```

Comment:
License?

Reply:
scancode is ignoring .txt files.   
most of the files in this directory are missing license headers 😿

Reply:
Oops.

Reply:
added.